frontend/src/app/style-guide/style-guide.component.html

```diff
@@ -0,0 +1,41 @@
+<section class="usa-section">
+  <div class="usa-grid">
+    <h1>Style Guide</h1>
+    <hr>
+    <h1>Heading 1</h1>
+    <h2>Heading 2</h2>
+    <h3>Heading 3</h3>
+    <h4>Heading 4</h4>
+    <h5>Heading 5</h5>
+    <h6>Heading 6</h6>
+    <p class="usa-font-lead">This is a paragraph lead. </p>
+    <p>This is a paragraph. This is a paragraph. This is a paragraph. This is a paragraph. This is a paragraph. This is a paragraph. This is a paragraph. This is a paragraph. This is a paragraph. This is a paragraph. </p>
+    <p>This is a paragraph. This is a paragraph. This is a paragraph. This is a paragraph. This is a paragraph. This is a paragraph. This is a paragraph. This is a paragraph. This is a paragraph. This is a paragraph. </p>
+
+    <a href="#">This is an anchor</a>
+
+    <h2>Buttons</h2>
+    <button class="usa-button">Primary</button> <span> - used for page navigation, hold applications</span><br />
```

Comment:
FYI, there are still some lingering `fs-button-green`s hidden throughout the site.